ui/plugin-system/src/components/PluginSpecEditor.tsx

```diff
@@ -42,11 +40,9 @@ export function PluginSpecEditor(props: PluginSpecEditorProps) {
   }
 
   if (pluginType === 'Panel') {
```

Comment:
Does the code in this `Panel` conditional still get used at all or is it all handled by `PanelSpecEditor` now?

Reply:
Bump on this question. Guessing it got lost in the sea of other feedback.

Reply:
oops, updated this to throw an error instead since it should be handled by `PanelSpecEditor` now